DESCRIPTION

```diff
@@ -1,13 +1,10 @@
 Package: SmaRP
 Title: SmaRP: Smart Retirement Planning
 Version: 1.3.0-9000
-Authors@R: c(person("Gabriel", "Foix", role = c("aut", "cre"),
-                    email = "gabriel.foix@mirai-solutions.com"),
-             person("Francesca", "Vitalini", role = c("aut"),
-                    email = "francesca.vitalini@mirai-solutions.com"),
-             person("Nicoletta", "Farabullini", role = c("aut"),
-                    email = "nicoletta.farabullini@mirai-solutions.com"),
-             person("Riccardo", "Porreca", role = c("ctb"),
+Authors@R: c(person("Gabriel", "Foix", role = c("aut")),
+             person("Francesca", "Vitalini", role = c("aut")),
+             person("Nicoletta", "Farabullini", role = c("aut")),
+             person("Riccardo", "Porreca", role = c("ctb", "cre"),
                     email = "riccardo.porreca@mirai-solutions.com"),
              person("Mirai Solutions GmbH", role = c("ctb", "cph"),
```

Comment:
Let's make the maintainer an author, which makes more sense in general (+ the contributor role was assigned very early before I stated "authoring" the app quite a bit).
Also, Mirai should just be the copyright holder

```suggestion
             person("Riccardo", "Porreca", role = c("aut", "cre"),
                    email = "riccardo.porreca@mirai-solutions.com"),
             person("Mirai Solutions GmbH", role = "cph",
```